docs/index.yml

```
@@ -38,6 +38,12 @@ landingContent:
             url: project/index.yml
           - text: Word add-ins documentation
             url: word/index.yml
+      - linkListType: learn
+        links: 
+          - text: Introduction to Office client customization with add-ins
+            url: //docs.microsoft.com/learn/modules/intro-office-add-ins/
```

Since this is a docs.microsoft.com link, you don't need anything before the ".com".
```suggestion
            url: /learn/modules/intro-office-add-ins/
```